.github/workflows/contracts-test.yaml

```diff
@@ -32,6 +32,7 @@ jobs:
         working-directory: clients/py
         run: |
           python3 -mpip install --user -r requirements.txt
+          python3 -mpip install --user -r requirements.dev.txt
```

Comment:
If the Python tests have been moved into `ci-test`, we can remove the python tests from `contracts-test`

Reply:
Agreed, removed python tests from contract-test.yaml in the latest commit.